conda-forge.yml

```diff
@@ -1,6 +1,8 @@
+azure:
+  store_build_artifacts: true
 bot:
   abi_migration_branches:
-    - v3.5.x
+  - v3.5.x
```

Comment:
Ha! @xylar

Reply:
Hmm, this is what I had wanted all along but I don't think the spacing was the problem anyway.